go.mod

```
@@ -3,19 +3,19 @@ module sigs.k8s.io/controller-runtime
 go 1.17
 
 require (
-	github.com/evanphx/json-patch v4.12.0+incompatible
-	github.com/fsnotify/fsnotify v1.5.1
-	github.com/go-logr/logr v1.2.0
-	github.com/go-logr/zapr v1.2.0
-	github.com/google/go-cmp v0.5.5
+	github.com/evanphx/json-patch v5.6.0+incompatible
```

```suggestion
	github.com/evanphx/json-patch/v5 v5.6.0
```
I think you have to import it like this. They started using module path correctly with one of the latest releases IIRC (+ update the imports accordingly)

---

Thanks, I wasn't aware. I added a commit. :)

---

Thx!

---

I had this issue in CAPI when bumping. I think otherwise you will end up with the code in the root folder of the repo which is afaik some v4 version :)